rust/src/test.rs

```diff
@@ -8,12 +8,14 @@ pub use crate::flags::*;
 /// Function which indicates if the test should be skipped by returning an error.
 pub type Guard = fn(&Config, &Path) -> Result<(), anyhow::Error>;
 
+#[derive(Clone, Copy)]
 pub enum TestFn {
     Serialized(fn(&mut SerializedTestContext)),
     NonSerialized(fn(&mut TestContext)),
 }
 
 /// A single minimal test case.
+#[derive(Clone, Copy)]
```

Comment:
Why do we want to derive these traits here, especially Copy?

Reply:
It's all about the `trim_start_matches`.  In order to make that work, I needed TestCase to be `Clone`, in an earlier version of this PR.  And I added `Copy` too under the general maxim that "anything that can be Copy should be".  But now I see that Clone isn't needed in the final version of the PR.  I can remove them.